plans/tier0.fmf

```diff
@@ -1,6 +1,10 @@
 discover+:
     tier: 0
 
+/check_cve:
```

Comment:
We should be able to pass the CVE number here too, right? 
```suggestion
/check_cve_2022_1662:
```

Reply:
It's more of a general check for all CVEs we'll ever have I think. Since this is the first one it is just a bit empty atm

Reply:
Ah, I see! Makes sense from this point of view, then!